app/controllers/occupation_standards_controller.rb

```diff
@@ -3,6 +3,10 @@ def index
     @page_title = "Occupations"
 
     if Flipper.enabled?(:use_elasticsearch_for_search)
+      if search_term_params[:q].present?
+        refine_search_params
```

Comment:
@FerPerales the only thing I'm a little worried about with this is if someone enters "15" and hits return. There could be matches against ONET "15-xxxx" and also RAPIDS code "15XX", and it would take the first hit and then grab that ONET code to limit on, thereby excluding results from one or the other. Should we only refine the search if the first character is a letter?

Reply:
@littleforest good point. Should we match only when string is "15-" and not only "15"?

Reply:
What about not worrying about the numbers, and only doing search refinement when someone enters letters for search? That way if someone enters "15", they would still get records back for ONET codes 15-xxxx and RAPIDS codes 15XX.